lib/new_relixir/agent.ex

```diff
@@ -31,9 +31,9 @@ defmodule NewRelixir.Agent do
 
         struct["return_value"]
       {:ok, 503, _} ->
-        raise RuntimeError.message("newrelic_down")
+        raise "newrelic_down"
```

Comment:
This fixes 
```elixir
lib/new_relixir/agent.ex:34: 
The call 'Elixir.RuntimeError':message(<<_:104>>) will never return since it differs in the 1st argument from the success typing arguments: (atom() | #{'message':=_, _=>_})
```
By raising the sting message directly we will already raise a `RuntimeError`